spec/controller/controller.test.js

```diff
@@ -0,0 +1,124 @@
+const _ = require('lodash');
+const chai = require('chai');
+const sinonChai = require('sinon-chai');
+const sinon = require('sinon');
+require('sinon-mongoose');
+const controller = require('../../server/controller.js');
+const { mockReq, mockRes } = require('sinon-express-mock');
+const Event = require('../../database/Event.js');
+const Org = require('../../database/Org.js');
+
+chai.use(sinonChai); // TODO: change from using Sinon's assertion style
+
+// TODO: this will probably need to change when benchmarking and stress testing the db ...
+const MAX_NUMBER_OF_EVENTS = 100;
+let sampleReq = {
+  params: {
+    eventId: MAX_NUMBER_OF_EVENTS
+  }
+}
+
+let eventMock;
+let orgMock;
+const returnedEvent = {
+  title: 'Testing API integration',
```

Comment:
This implies that they're integration tests with the database not unit tests with mocked endpoints/data